image/v1/types.go

```diff
@@ -60,7 +60,30 @@ type Image struct {
 	// DockerImageManifestMediaType specifies the mediaType of manifest. This is a part of manifest schema v2.
 	DockerImageManifestMediaType string `json:"dockerImageManifestMediaType,omitempty" protobuf:"bytes,9,opt,name=dockerImageManifestMediaType"`
 	// DockerImageConfig is a JSON blob that the runtime uses to set up the container. This is a part of manifest schema v2.
+	// Will not be set when the image represents a manifest list.
 	DockerImageConfig string `json:"dockerImageConfig,omitempty" protobuf:"bytes,10,opt,name=dockerImageConfig"`
+	// DockerImageManifests holds information about sub-manifests when the image represents a manifest list.
+	// When this field is present, no DockerImageLayers should be specified.
+	DockerImageManifests []ImageManifest `json:"dockerImageManifests,omitempty" protobuf:"bytes,11,rep,name=dockerImageManifests"`
```

Comment:
`validateImage` in your openshift-apiserver PR appears to be missing validation rules

Reply:
I just tested it and there's already validation of types in place (i.e a string only accepts strings, etc), so I don't think we need further validation.